touchscreen.lua

```diff
@@ -101,6 +97,12 @@ local function process_command(meta, data, msg)
 			table.remove(data, index)
 		end
 
+	elseif cmd == "remove_without_resize" then
+		local index = tonumber(msg.index)
+		if index and data[index] then
+			data[index] = ""
+		end
```

Comment:
I prefer the command to be `delete`, `remove_without_resize` is too long. Also, since nil values are no longer an issue, you should the index to nil to save space.
```suggestion
	elseif cmd == "delete" then
		local index = tonumber(msg.index)
		if index and data[index] then
			data[index] = nil
		end
```